Cargo.toml

```diff
@@ -44,3 +44,4 @@ ndarray = "0.12.1"
 sprs = "0.6.3"
 unicode-segmentation = "1.2.1"
 hashbrown = "0.1"
+float-cmp = "0.4.0" #for asserting floating point differences in test
```

Comment:
Maybe we could put this under the `[dev-dependencies]` section, as it's only needed for tests?

Nit; also maybe adding a space after the `#` for readability if you don't mind.

Reply:
For sure. Will do. I'll look around too for something like `numpy.testing.assert_allclose`.